bin/genesis

```diff
@@ -1082,7 +1086,7 @@ EOF
 		for file in ${DEPLOYMENT_ROOT}/.env_hooks/*; do
 			if [[ -f ${file} && -x ${file} ]]; then
 				echo "Running env setup hook: ${file}"
-				export VAULT_PREFIX DEPLOYMENT_NAME DEPLOYMENT_SITE
+				export VAULT_PREFIX DEPLOYMENT_NAME DEPLOYMENT_SITE DEPLOYMENT_ENVIRONMENT DEPLOYMENT_ROOT ENVIRONMENT_ROOT=${DEPLOYMENT_ROOT}/${DEPLOYMENT_SITE}/${DEPLOYMENT_ENVIRONMENT}
```

Comment:
Do bosh-init environments get the `DEPLOYMENT_ENVIRONMENT` and `DEPLOYMENT_SITE` env vars when you are in the middle of creating a new env?